### ISSUE TITLE
TextColors$1 cannot be cast to SpongeTextColor

### ISSUE THREAD

**Issue by Nunnery** [SpongePowered/Sponge#347]
Running Sponge (coremod) version 1.8-1499-2.1DEV-634, fresh install. Ran "sponge plugins" in the console and was greeted with an error. Need any more information, ping me and I'll get it back to you.

```
sponge plugins
[00:32:01] [Server thread/INFO]: Error occurred while executing command: org.spongepowered.api.text.format.TextColors$1 cannot be cast to org.spongepowered.common.text.format.SpongeTextColor
[00:32:01] [Server thread/ERROR] [Sponge]: Error occurred while executing command 'sponge plugins' for source net.minecraft.server.dedicated.DedicatedServer@6e1f76ab: org.spongepowered.api.text.format.TextColors$1 cannot be cast to org.spongepowered.common.text.format.SpongeTextColor
java.lang.ClassCastException: org.spongepowered.api.text.format.TextColors$1 cannot be cast to org.spongepowered.common.text.format.SpongeTextColor
        at org.spongepowered.api.text.Text.initializeComponent(Text.java:78) ~[Text.class:1.8-1499-2.1DEV-634+spongeproject-ci-b634.git-38a5fb2d71d332808348526a2a8c3beffb2c18c4]
        at org.spongepowered.api.text.Text.getHandle(Text.java:111) ~[Text.class:1.8-1499-2.1DEV-634+spongeproject-ci-b634.git-38a5fb2d71d332808348526a2a8c3beffb2c18c4]
        at org.spongepowered.api.text.Text.toPlain(Text.java:121) ~[Text.class:1.8-1499-2.1DEV-634+spongeproject-ci-b634.git-38a5fb2d71d332808348526a2a8c3beffb2c18c4]
        at org.spongepowered.common.text.SpongeTextFactory.toPlain(SpongeTextFactory.java:61) ~[SpongeTextFactory.class:1.8-1499-2.1DEV-634+spongeproject-ci-b634.git-38a5fb2d71d332808348526a2a8c3beffb2c18c4]
        at org.spongepowered.common.text.SpongeTextFactory.toPlain(SpongeTextFactory.java:56) ~[SpongeTextFactory.class:1.8-1499-2.1DEV-634+spongeproject-ci-b634.git-38a5fb2d71d332808348526a2a8c3beffb2c18c4]
        at org.spongepowered.api.text.Texts.toPlain(Texts.java:547) ~[Texts.class:1.8-1499-2.1DEV-634+spongeproject-ci-b634.git-38a5fb2d71d332808348526a2a8c3beffb2c18c4]
        at org.spongepowered.api.util.command.args.CommandElement.getUntranslatedKey(CommandElement.java:74) ~[CommandElement.class:1.8-1499-2.1DEV-634+spongeproject-ci-b634.git-38a5fb2d71d332808348526a2a8c3beffb2c18c4]
        at org.spongepowered.api.util.command.args.CommandElement.parse(CommandElement.java:93) ~[CommandElement.class:1.8-1499-2.1DEV-634+spongeproject-ci-b634.git-38a5fb2d71d332808348526a2a8c3beffb2c18c4]
        at org.spongepowered.api.util.command.args.ChildCommandElementExecutor.parse(ChildCommandElementExecutor.java:140) ~[ChildCommandElementExecutor.class:1.8-1499-2.1DEV-634+spongeproject-ci-b634.git-38a5fb2d71d332808348526a2a8c3beffb2c18c4]
        at org.spongepowered.api.util.command.args.GenericArguments$FirstParsingCommandElement.parse(GenericArguments.java:364) ~[GenericArguments$FirstParsingCommandElement.class:1.8-1499-2.1DEV-634+spongeproject-ci-b634.git-38a5fb2d71d332808348526a2a8c3beffb2c18c4]
        at org.spongepowered.api.util.command.spec.CommandSpec.populateContext(CommandSpec.java:287) ~[CommandSpec.class:1.8-1499-2.1DEV-634+spongeproject-ci-b634.git-38a5fb2d71d332808348526a2a8c3beffb2c18c4]
        at org.spongepowered.api.util.command.spec.CommandSpec.process(CommandSpec.java:333) ~[CommandSpec.class:1.8-1499-2.1DEV-634+spongeproject-ci-b634.git-38a5fb2d71d332808348526a2a8c3beffb2c18c4]
        at org.spongepowered.api.util.command.dispatcher.SimpleDispatcher.process(SimpleDispatcher.java:345) ~[SimpleDispatcher.class:1.8-1499-2.1DEV-634+spongeproject-ci-b634.git-38a5fb2d71d332808348526a2a8c3beffb2c18c4]
        at org.spongepowered.api.service.command.SimpleCommandService.process(SimpleCommandService.java:251) [SimpleCommandService.class:1.8-1499-2.1DEV-634+spongeproject-ci-b634.git-38a5fb2d71d332808348526a2a8c3beffb2c18c4]
        at net.minecraft.command.ServerCommandManager.func_71556_a(SourceFile:80) [cl.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71333_ah(DedicatedServer.java:400) [po.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:365) [po.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:598) [MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:478) [MinecraftServer.class:?]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_60]
```

**Comment by bloodmc** [comment]
Update to latest build, that should be fixed.

**Comment by Nunnery** [comment]
Verified fixed in version 1.8-1499-2.1DEV-645.
